app/components/conceptAdd/conceptAdd.controller.js

```diff
@@ -113,7 +119,21 @@
 			validateForm()
 		}
 		function postConcept(){
-			vm.result = conceptsService.postConcept(vm.concept, vm.localizedConcepts);
+			conceptsService.postConcept(vm.concept, vm.localizedConcepts).then(function(result){
+				if(angular.isDefined(result)&&result.success){
+					$location.path('/concept/').search({added : result.message});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'vm'.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2347615092/issues/source?bid=3196969&fileBranchId=3209439#l126)